pkg/monitor/cluster/arooperatorheartbeat.go

```diff
@@ -0,0 +1,28 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"strconv"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitAroOperatorHeartbeat(ctx context.Context) error {
+	aroDeployments, err := mon.cli.AppsV1().Deployments("openshift-azure-operator").List(metav1.ListOptions{})
+
+	if err != nil {
+		return err
+	}
+
+	for _, d := range aroDeployments.Items {
+		mon.emitGauge("arooperator.heartbeat", 1, map[string]string{
+			"name":    d.Name,
+			"healthy": strconv.FormatBool(d.Status.UnavailableReplicas == 0),
```

Comment:
@mjudeikis and I discussed, unless there is a strong reason, we think it makes sense to standardise and use ready.DeploymentIsReady here.  We are OK with the concept that this metric will blip, e.g. during operator upgrade or during master node draining and rebooting - the corresponding alert should not be so sensitive that this is a problem.

Reply:
Done.